src/Sylius/Bundle/CoreBundle/Form/Type/User/AdminUserType.php

```diff
@@ -28,8 +29,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         parent::buildForm($builder, $options);
 
         $builder
-            ->add('firstName')
-            ->add('lastName')
+            ->add('firstName', TextType::class, [
+            	'label' => 'sylius.ui.first_name',
```

Comment:
@pjedrzejewski Are you sure? I see `sylius.form.user.first_name` but no `sylius.form.admin_user.first_name`.